tests/Unit/NumericalAlgorithms/Interpolation/InterpolationTargetTestHelpers.hpp

```diff
@@ -141,8 +141,7 @@ struct mock_interpolator {
 template <typename MetaVariables, typename DomainCreator,
           typename InterpolationTargetOption, typename BlockCoordHolder>
 void test_interpolation_target(
-    const DomainCreator& domain_creator,
-    const InterpolationTargetOption& options,
+    const DomainCreator& domain_creator, InterpolationTargetOption&& options,
```

Comment:
take by value

Reply:
Done.  This required other changes (std::moves and IWYU stuff); I squashed that into the last commit.